### DISCUSSION TITLE
How to use DynamicResource in XAML in Windows App SDK

### DISCUSSION THREAD

**Discussion by KishlayaSharma** [microsoft/WindowsAppSDK#3689] [Q&A]
Hello,
I am getting errors when trying to use DynamicResource in XAML like
`<Button Background="{DynamicResource ButtonExitBackgroundColor}"  Content="Dynamically Styled"/>`

I am trying to follow this tutorial [[Link](https://www.c-sharpcorner.com/article/static-dynamic-resources-in-wpf/)]

![image](https://github.com/microsoft/WindowsAppSDK/assets/39042276/9b40331a-fc74-4c22-a3bd-120952acfde7)

**Comment by ghost1372** [comment]
DynamicResource is for WPF, in winui 3 you should use staticResource or ThemeResource
